packages/veritone-react-common/src/components/FormBuilder/FormBuilder.js

```diff
@@ -93,15 +94,15 @@ function FormBuilderComponent({
           />
         )}
       </div>
-      <div className={styles['blocks-preview']}>
+      <div className={cx(styles['blocks-preview'], classes.previewContainer)}>
         {
           form.length > 0 && (
             <Typography className={cx(styles['form-preview-title'])}>
               Form preview
             </Typography>
           )
         }
-        <div ref={drop}>
+        <div ref={drop} className={classes.previewContent}>
```

Comment:
Can we be consistent with className thing? 
I mean `styles['blocks-preview'], classes.previewContainer`

Reply:
I don't think we should as styles comes from scss file and classes comes from js. They come from different convention

Reply:
resolved!